en/docs/get-started/development-kickstart.md

```diff
@@ -616,44 +616,18 @@ Let's start the mock back-end services for this use case:
 
 ### Invoke the Healthcare service
 
-There are two ways to invoke the service:
+Use a REST client of your choice to send a request to the integration service. In this tutorial we are using cURL.
```

Comment:
Shall we use `curl` instead of `cURL`? https://curl.se/docs/

Reply:
+1